### PR TITLE
Foolnidiot card

### PR DESCRIPTION
**Pull request by foolnidiot**

### CODE REVIEW

**Comment by foolnidiot** [comment]
Added my card

File: index.html [hunk 1]
```diff
@@ -114,7 +114,36 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
-
+<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (dilshankarunarathne) [inline index.html:117]: Please change this comment 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` 
into `<!-- ________ Your_Name Contributor card START ________ -->`

File: index.html [hunk 2]
```diff
@@ -114,7 +114,36 @@ <h2>A project for first-time contributions</h2>
...
+      </li>
+      <li>
+        <a href="https://www.reddit.com/r/programming/" target="_blank" title="Third resource">
+          Reddit Programming
+        </a>
+      </li>
+    </ul>
+  </div>
+</div>
+<!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (dilshankarunarathne) [inline index.html:146]: Please change this comment as well. 
`<!-- ________*END TEMPLATE* Contributor card END ________ -->` 
to `<!-- ________ Your_Name Contributor card END ________ -->`

File: index.html [hunk 3]
```diff
@@ -114,7 +114,36 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
```
Comment (dilshankarunarathne) [inline index.html:116]: Please leave an empty line between the template and your code.

File: index.html [hunk 4]
```diff
@@ -114,7 +114,36 @@ <h2>A project for first-time contributions</h2>
...
+      <li>
+        <a href="https://www.reddit.com/r/programming/" target="_blank" title="Third resource">
+          Reddit Programming
+        </a>
+      </li>
+    </ul>
+  </div>
+</div>
+<!-- ________*END TEMPLATE* Contributor card END ________  -->
         <!-- ________ ComradeGuero's card START ________  -->
```
Comment (dilshankarunarathne) [inline index.html:147]: Please leave an empty line after your code.

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
Hello @foolnidiot, thank you for contributing to this project. ❤
But there are a couple of small changes that need to be done before we go ahead and merge your code. Please change these two comments, because they are very helpful for everyone to identify each card. 
Leaving two blank lines around your code is also important. 
Once you've made those changes, please get back to us. We will be able to review and merge your pull request in a flash.

**Review by dilshankarunarathne — APPROVED** [review]

**Comment by dilshankarunarathne** [comment]
Congratulations 🎉❤
Your pull request was merged. 
You will be able to see your card on the [project website](https://syknapse.github.io/Contribute-To-This-Project/).

**Pull request merged**